src/dialog/dialog.scss

```diff
@@ -29,6 +29,11 @@
   }
 }
 
+@mixin iui-dialog-backdrop-relative {
```

Comment:
Why it can't go by default to draggable dialog?

Reply:
But how react api will look like?

Reply:
option to choose either relative to page or relative to container?